app/src/main/res/layout/activity_main.xml

```diff
@@ -16,87 +16,84 @@
         android:text="@string/drag_me"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"/>
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
         android:id="@+id/first"
         style="@style/AppTheme.CircleTextView"
-        android:layout_width="@dimen/circle_size"
-        android:layout_height="@dimen/circle_size"
+        android:layout_width="@dimen/other_circles_size"
+        android:layout_height="@dimen/other_circles_size"
         android:layout_marginTop="@dimen/circle_distance"
         android:backgroundTint="#388e3c"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/drag"/>
```

Comment:
I think you don't need to (and shouldn't - to avoid confusion) ask Android to create a new `drag` `id` here since it has already been created above in `drag` view declaration

Reply:
It doesn't have a plus sign, so I think it's not creating a new ```drag id```.

Reply:
My bad, I misread it